src/components/ViolationForm.js

```diff
@@ -3,6 +3,8 @@ import styled from 'styled-components'
 import React, { useEffect, useState } from 'react'
 import api from '../utils/api'
 import { SectionSelector } from './sectionSelector/SectionSelector'
+import UploadPhotos from './UploadPhotos'
+import { saveImages, convertImagesToBase64 } from '../utils/uploadPhotosHelper'
 
 const CommentFormStyle = styled.form`
```

Comment:
Знам, че не е част от твоята работа, но ако може да вмъкнеш този edit на CSS-a, за да е по-добре подравнена формата. Освен това е хубаво да я погледнем на mobile, защото е доста малка.

```suggestion
const CommentFormStyle = styled.form`
  width: 100%;

  .errorMsg {
    color: red;
  }

  textarea {
    width: 80%;
    height: 50px;
    padding: 20px;
    margin-left: 5px;
    margin-bottom: 10px;
  }

  input[type='radio'] {
    margin: 5px;
    vertical-align: middle;
  }

  .inputLabel {
    display: block;
    margin-top: 10px;
    margin-left: 5px;
    padding: 5px;
  }

  input[type='text'] {
    width: 80%;
    font-size: 18px;
    padding: 10px;
    border: 1px solid #eee;
    margin-bottom: 10px;
    box-sizing: border-box;
    margin-left: 5px;
  }

  select {
    margin-left: 5px;
    padding: 5px;
  }

  .successfulMessage {
    color: green;
  }

  .unsuccessfulMessage {
    color: red;
  }

  button {
    padding: 10px;
    margin: 20px 5px;
  }
`
```